itemcrate/game-jam-pgh-2018
# Adds leaderboard and initials scenes

# Description
Adds scenes for the initials input and leaderboard scenes.

# Testing
0. Add leaderboard.json! Here's some dummy data:

`{"leaderboard":[{"name":"D7F","score":728},{"name":"CAB","score":412},{"name":"DAB","score":190},{"name":"COK","score":25},{"name":"ASS","score":0},{"name":"E7A","score":0},{"name":"D8E","score":0},{"name":"GG6","score":0},{"name":"EEF","score":0},{"name":"EF7","score":0}]}`


1. Start the game and collide with enemies until dead.
2. Score will be randomized. If higher than the lowest score on the leaderboard, the initials scene should tween visibility. Otherwise, the leaderboard will fade in.
3. If on the initials scene, enter your initials. A blinking cursor should indicate your current initial space.
4. Once three characters have been added, the initials scene should fade out and be replaced by the leaderboard
